__mock__/mockDynamoDb.js

```diff
@@ -13,83 +24,90 @@ class MockDynamoDb {
         this.returnObject = null;
         this.deleteInput = null;
         this.putInput = null;
-    }
-    get(params) {
+        this.updateInputs = [];
+    };
+    MockDynamoDb.prototype.get = function (params) {
+        var _this = this;
         return {
-            promise: () => {
-                return new Promise((resolve, reject) => {
+            promise: function () {
+                return new Promise(function (resolve, reject) {
                     resolve({
                         '$response': {
-                            error: this.error,
+                            error: _this.error
```

Comment:
Typescript resolves this, where javascript wouldn't.  There's no need for the _this resolution